docs/creating_dataset.md

```diff
@@ -12,30 +14,33 @@ After creating the dataset, you can also [analyze](#analysing-segmentation-datas
+
+* Each subject has one or more scans, and one or more segmentation masks. There should be one segmentation mask for
    each ground truth structure (anatomical structure that the model should segment)
- * For convenience, scans and ground truth masks for different subjects can live in separate folders, but that's not a must.
- * Inside the root folder for the dataset, there should be a file `dataset.csv`, containing the following fields
+* For convenience, scans and ground truth masks for different subjects can live in separate folders, but that's not a must.
+* Inside the root folder for the dataset, there should be a file `dataset.csv`, containing the following fields
  at minimum:
-    * `subject`: A unique positive integer assigned to every patient
-    * `channel`: The imaging channel or ground truth structure described by this row.
-    * `filePath`: Path to the file for this scan or structure. We support nifti (nii, nii.gz), numpy (npy, npz) and hdf5(h5).
-        * For HDF5 path suffix with | separator
-            * For images <path>|<dataset_name>|<channel index>
-            * For segmentation binary <path>|<dataset_name>|<channel index>
-            * For segmentation multimap <path>|<dataset_name>|<channel index>|<multimap value>
-                * Multimaps are encoded as 0=background and integers for each class.
-            * The expected dimensions: (channel, Z, Y, X)
-        * For numpy or nifti just the expected format is just the path to the files.
-            * For images can be encoded as float32 with dimensions (X, Y, Z)
-            * For segmentations should be encoded as binary masks with dimensions (X, Y, Z)
+  * `subject`: A unique positive integer assigned to every patient
+  * `channel`: The imaging channel or ground truth structure described by this row.
+  * `filePath`: Path to the file for this scan or structure. We support nifti (`.nii`, `.nii.gz` extensions), numpy (`.npy`, `.npz`) and hdf5(`.h5`).
+    * For HDF5 files, you need set the the actual file path, and specify the HDF5 dataset name and channel as follows with `|` as a separator:
+      * For images: `<path>|<dataset_name>|<channel index>`
+      * For segmentations that are provided as binary maps: `<path>|<dataset_name>|<channel index>`
+      * For segmentations that are given as multimaps: `<path>|<dataset_name>|<channel index>|<multimap value>`
+        * Multimaps are encoded as 0=background and integers for each class.
+      * The expected dimensions: (channel, Z, Y, X)
+    * For numpy or nifti just the expected format is just the path to the files.
+      * Images can be encoded as float32 with dimensions (X, Y, Z)
+      * Segmentations should be encoded as binary masks with dimensions (X, Y, Z). There should be one binary mask per
+        ground truth structure. The arrays should contain 1 for all voxels that belong to the structure, and 0 for all other voxels.
+        You can save those to nifti by working with numpy `uint8` arrays.
```

Comment:
This might not be very clear. Does it mean that the ground truth label maps _must_ have data type `uint8`?